### PR TITLE
More Alt titles

### PR DESCRIPTION
**Pull request by Tename**
Adds the follow alt titles and renames Trainee Medical Technician to Medical Trainee

**Engineer**
Life Support Technician
**Medical Trainee**
Trainee Medical Technician
Nurse
**Scientist**
Circuitry Designer
Chemical Researcher

### CODE REVIEW

**Comment by Tename** [comment]
Trying to figure out why the Trainee Civs aren't working.

**Comment by Sparrowbird611** [comment]
Gonna want to hear the rationale behind this dispatch officer. It seems like a bad title IMO, since the BO isn't the 911 responder guy.

The rest I like though.

**Comment by Roland410** [comment]
Gonna go with a big NO on civilian trainees and BOs. Contractors are not meant to be trained/learning shit on ship as they are supposed to be experts. Contractor BO also makes no sense.

**Comment by Tename** [comment]
> 
> 
> Gonna go with a big NO on civilian trainees and BOs. Contractors are not meant to be trained/learning shit on ship as they are supposed to be experts. Contractor BO also makes no sense.

This doesnt add civilian BOs only Medical Trainees, I'll limit them back to Fleet and SCG though, I still think Civs should be allowed to be trainee. But I'll probably put it in a different PR

File: maps/torch/job/medical_jobs.dm [hunk 1]
```diff
@@ -95,16 +95,19 @@
...
 	allowed_branches = list(
 		/datum/mil_branch/expeditionary_corps,
-		/datum/mil_branch/fleet = /decl/hierarchy/outfit/job/torch/crew/medical/doctor/fleet
+		/datum/mil_branch/fleet = /decl/hierarchy/outfit/job/torch/crew/medical/doctor/fleet,
 	)
+
 	allowed_ranks = list(
 		/datum/mil_rank/ec/e3,
-		/datum/mil_rank/fleet/e2
+		/datum/mil_rank/fleet/e2,
```
Comment (BurgerLUA) [inline maps/torch/job/medical_jobs.dm:110]: This didn't need a comma. Several of these additions exist throughout this PR.

**Comment by Tename** [comment]
Fix it, but Github keeps going down, so it isn't updating, will check again in an hour.

**Comment by N8-Toe** [comment]
Nurse doesn't imply trainee. and we have the "resident" alt title for new doctors/doctors in training. I think that change is un-needed and will lead to confusion

**Comment by Tename** [comment]
Placing nurse as a normal physician would make them officer level

**Comment by Roland410** [comment]
I'd give nurse to MTs and Nurse trainee for trainee.

**Comment by Tename** [comment]
> 
> 
> I'd give nurse to MTs and Nurse trainee for trainee.

Alright, that works I guess, I'll get to it when I have a chance

File: maps/torch/job/medical_jobs.dm [hunk 2]
```diff
@@ -95,13 +96,16 @@
 	selection_color = "#013d3b"
 	minimum_character_age = list(SPECIES_HUMAN = 18)
 	alt_titles = list(
-		"Corpsman Trainee")
+		"Trainee Medical Technician",
```
Comment (Roland410) [inline maps/torch/job/medical_jobs.dm:99]:
```suggestion
		"Medical Technician Trainee",
```

File: maps/torch/job/medical_jobs.dm [hunk 3]
```diff
@@ -95,13 +96,16 @@
...
+		"Corpsman Trainee",
+		"Trainee Nurse")
```
Comment (Roland410) [inline maps/torch/job/medical_jobs.dm:101]:
```suggestion
		"Nurse Trainee")
```

**Pull request merged**